File: src/main/java/co/utb/softeng/moviesapp/entities/Actor.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package co.utb.softeng.moviesapp.entities;

import java.io.Serializable;
import java.util.Date;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

/**
 *
 * @author Juan Mantilla
 */
@Entity
@Table(name = "ACTORS")
public class Actor implements Serializable {
    
    @Id
    @GeneratedValue
    private Long id;
    private String name;
    private String lastName;
    private String gender;
    @Temporal(value=TemporalType.DATE)
    private Date nacimiento;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }
    
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
    
     public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }
    
     public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public Date getNacimiento() {
        return nacimiento;
    }

    public void setNacimiento(Date nacimiento) {
        this.nacimiento = nacimiento;
    }
   
    
    
    
}

File: src/main/webapp/resources/assets/js/app/actor.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

$.ajax({
    type: 'GET',
    url: apiUrl + 'actor/' + actorId
}).done(function (actor) {
    var html = '';

    html += "<div class='actor'>Name:<a href='actors/"+actor.id+"'>" + actor.name + " "+ actor.lastName+"</a><br/>";
    html += "Gender: " + actor.gender + "</div>";
    html += "Date of birth: " + actor.nacimiento + "</div>";
    html += "<br/> <h1>Movies:</h1> <div id='actorsDiv'>";
    _.each(actor.movies, function (movie) {
        html += "<div class='movie'>Name: <a href='"+apiUrl+"movies/"+movie.id+"'>" + movie.name + "</a><br/>";
        html += "Genre: " + movie.genre + "</div>";
        html += "Release Date: " + movie.releaseDate + "</div>";

    });
    html += "</div>";


    $('#actorDiv').html(html);
});

